arcsi/api/data.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime, timedelta
+
+from flask import jsonify, make_response, request
+from flask_security import roles_required
+
+from . import arcsi
+from arcsi.model.item import Item
+
+headers = {"Content-Type": "application/json"}
+
+@arcsi.route("/data/episodes_uploaded_daily", methods=["POST"])
+@roles_required("guest")
+def uploaded_episodes_in_last_x_days():
+    date_metadata = request.form.to_dict()
+    from_date = date_metadata['from_date']
+    given_date = datetime.strptime(from_date, '%Y-%m-%d')
+    if (given_date > datetime.today()):
+        return make_response("Back to the future?", 418, headers)
+    last_days = int(date_metadata['last_days'])
+    if (last_days < 0):
+        return make_response("Please, be postive!", 400, headers)
+    episodes_count = Item.query.filter(
+        given_date - timedelta(days=last_days) <= Item.play_date
+        ).filter(Item.play_date <= datetime.today()
+        ).filter(Item.archived == True).count()
```

Comment:
@tuz666 it seems we don't count live episods that we haven't archived yet. I think we should, cause they have been aired too.